### PR TITLE
Emphasise keywords and rephrase awkward sentences in UG up to Bill command

### PR DESCRIPTION
**Pull request by JustinPeng13**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -200,39 +195,39 @@ This command displays a list of all guests in GuestBook.
 
 This command edits an existing guest in GuestBook.
 
-The guest specified at the `INDEX` is the guest you wish to edit. The existing values will be updated to the values
-that you have inputted. The values that you did not specify to edit will remain the same.
+The guest specified at the `INDEX` is the guest you wish to edit. The existing values will be **updated** to the new values
+you provide. The values that you did not specify to edit will **remain unchanged**.
```
Comment (kangzongxian) [inline docs/UserGuide.md]: Should we bold the "did not specify" as well?

**Review by kangzongxian — CHANGES_REQUESTED** [review]
Looks good to me except for 1 potential minor change, great work!

File: docs/UserGuide.md (hunk 1 above)
Comment (JustinPeng13) [reply docs/UserGuide.md]: Changed!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -261,7 +256,7 @@ the rooms that have room clean statuses set to "no".
   ![result for 'find alex 99272758 03-68'](images/findAlex9927275803-68.png)
 
 #### Common mistakes:
-* Searching for guests using partial field characteristics e.g. `170` will not match `170.00`.
+* Searching for guests using partial field characteristics, e.g., `170` will not match `170.00`.
```
Comment (bryanhce) [inline docs/UserGuide.md]: Should this be E.g.?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -247,11 +242,11 @@ the rooms that have room clean statuses set to "no".
 #### Format: `find KEYWORD [MORE_KEYWORDS]`
 
 #### Constraints of parameters:
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only full words will be matched e.g. `Han` will not match `Hans`
+* The search is case-insensitive. E.g., `hans` will match `Hans`
+* The order of the keywords does not matter. E.g., `Hans Bo` will match `Bo Hans`
+* Only full words will be matched E.g., `Han` will not match `Hans`
 * Guests matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (bryanhce) [inline docs/UserGuide.md]: Should this be I.e.?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -44,65 +44,70 @@ This user guide provides in-depth descriptions of all the available features in
...
+3. Copy the file to the folder you want to use as the **home folder** for your GuestBook. <br> <br>
    ![Ui](images/GuestBookIcon.png) <br><br>
+
 4. Double-click on `guestbook.jar` to start the app. The GUI, similar to the image below, should pop up in a few seconds. 
-Note that the app contains sample data.<br><br>
+Note that the app contains **sample data**.<br><br>
    ![Ui](images/Ui.png) <br><br>
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press _Enter_ to execute it. E.g., typing **`help`** and pressing _Enter_ will open the help window.<br>
```
Comment (bryanhce) [inline docs/UserGuide.md:80]: I am unsure which kind of texts are bolded and which are italicised, can you share more?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -44,65 +44,70 @@ This user guide provides in-depth descriptions of all the available features in
...
-    * **`bill`** `3` `b/+30.35`: Adds 30.35 to the bill of the third guest.
+11. When a guest checks out of your hotel, you can remove the guest using the **`delete`** command.
+    * **`delete`** `3` : Deletes the third guest shown in the current list.
 
-11. When a guest checkouts, you might want to delete the guest. In that case, you can use the **`delete`** command.
-    * **`delete`**`3` : Deletes the third guest shown in the current list.
-
-12. To clear your GuestBook of all entries, you can use the **`clear`** command. Do be warn that it is irreversible, so use with caution!
+12. To clear your GuestBook of all entries, you can use the **`clear`** command.<br>
+<span style="background-color: yellow; color: black">**WARNING:** This action is **irreversible**. Use with caution.</span>
```
Comment (bryanhce) [inline docs/UserGuide.md:104]: This line is repeated under features, clearing all entries, should we have it here as well? Or do you think its necessary?

**Review by bryanhce — CHANGES_REQUESTED** [review]
some nits to fix

File: docs/UserGuide.md (hunk 2 above)
Comment (JustinPeng13) [reply docs/UserGuide.md]: I think no need because not at the start of a sentence

File: docs/UserGuide.md (hunk 3 above)
Comment (JustinPeng13) [reply docs/UserGuide.md]: Similarly, not at the start of a sentence so lowercase is okay

File: docs/UserGuide.md (hunk 4 above)
Comment (JustinPeng13) [reply docs/UserGuide.md:80]: Italics are used for special words while bold is used for important words

File: docs/UserGuide.md (hunk 5 above)
Comment (JustinPeng13) [reply docs/UserGuide.md:104]: I think it's good to leave it here as the user might accidentally unintentionally clear the data while following the tutorial!

**Review by kangzongxian — APPROVED** [review]
Looks good to me as the comments are addressed!

**Pull request merged**